### ISSUE TITLE
What is "regularization_loss" in train loss 

### ISSUE THREAD

**Issue by Runist** [zzh8829/yolov3-tf2#231]
I can't understand regularization_loss.How it work?
And when i print model.loss, it always get zeros.It is mean the regularization_loss is invalid?
` for batch, (images, labels) in enumerate(train_dataset):
                with tf.GradientTape() as tape:
                    outputs = model(images, training=True)
                    regularization_loss = tf.reduce_sum(model.losses)
                    pred_loss = []
                    for output, label, loss_fn in zip(outputs, labels, loss):
                        pred_loss.append(loss_fn(label, output))
                    total_loss = tf.reduce_sum(pred_loss) + regularization_loss`
